src/config/index.js

```diff
@@ -2,9 +2,9 @@ export const isDev = process.env.NODE_ENV !== 'production';
 export const DEV_PORT = 3200;
 export const BASE_BRANCH = 'master';
 
-export const HOST_URL = `https://opensource.hcltechsw.com`;
+export const HOST_URL = `https://honlamso.github.io`;
```

Comment:
I think changes in this file were for testing purpose. We can revert them.

Reply:
yes, thanks for pointing out